src/controllers/accomodation.controller.js

```diff
@@ -0,0 +1,62 @@
+import response from '../helpers/response.helper';
+import accomodation from '../services/accomodation.services';
+import UserServices from '../services/user.service';
+
+/**
+* Class for Travel Admins dealing with Accomodation
```

Comment:
@Jajabenit250 This is a class that models all functions to manage accommodation. It shouldn't be tied to the TravelAdmins only. Accommodation suppliers can also use functions that are provided in this class.